R/java_download.R

```diff
@@ -40,18 +40,19 @@ java_download <- function(
     cache_path <- file.path(temp_dir, "rJavaEnv_cache")
   }
 
-  rje_consent_check()
+  # rje_consent_check() # disabling consent check for now
   java_urls <- java_urls_load()
 
   valid_distributions <- names(java_urls)
   valid_platforms <- names(java_urls[[distribution]])
   valid_architectures <- names(java_urls[[distribution]][[platform]])
+  valid_versions <- 
```

Comment:
Is this supposed to be on its own line?

Reply:
nope. I wonder how r cmd check passed this. removed.